changes/721.feature.md

```diff
@@ -0,0 +1 @@
+Add commit history link between releases in release notes(`CHANGELOG_RELEASE.md`) by updating `scripts/extract-release-changelog.py`
```

Comment:
Sinse changelog is for informing "what" has changed - not "how" it's changed -, it'll be okay to skip which script you updated.
```suggestion
Add commit history link between releases in release notes(`CHANGELOG_RELEASE.md`).
```

Reply:
Thanks. I've updated changelog on the below commit!